samples/maps/demo/data-class-two-ranges/demo.js

```diff
@@ -165,16 +199,16 @@
             });
 
             // Initialize the chart
-            window.chart = new Highcharts.Map(options);
+            Highcharts.mapChart('container', options);
         },
 
         error: function () {
-            $('#container').html(
-                '<div class="loading">' +
-                '<i class="icon-frown icon-large"></i> ' +
-                '<p>Error loading data from Google Spreadsheets</p>' +
-                '</div>'
-            );
+            const container = document.getElementById('container');
+            container.innerHTML = `
```

Comment:
We don't have to set `container` variable, you can set `innerHTML` directly.

Reply:
Sure!